### ISSUE TITLE
Risk add command doesn't match actual rule.

### ISSUE THREAD

**Issue by netcons** [vel21ripn/nDPI#164]
## Describe the bug
When adding a risk filter the actual rule doesn't match the initial add command.
The first risk always becomes 0.

### Expected behavior
``` shell
iptables -A FORWARD  -m ndpi --all --risk 1,2,3 -j DROP
iptables -vnxL | grep risk
       0        0 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            ndpi risk 1,2,3 proto all
```
or

``` shell
iptables -A FORWARD  -m ndpi --all --risk 4,5,6 -j DROP
iptables -vnxL | grep risk
       0        0 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            ndpi risk 4,5,6 proto all
```

### Obtained behavior
``` shell
iptables -A FORWARD  -m ndpi --all --risk 1,2,3 -j DROP
iptables -vnxL | grep risk
       0        0 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            ndpi risk 0,2,3 proto all
```
or
``` shell
iptables -A FORWARD -m ndpi --all --risk 4,5,6 -j DROP
iptables -vnxL | grep risk
       0        0 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            ndpi risk 0,5,6 proto all
```

## nDPI Environment:
* OS name: [CentOS]
* OS version: [9]
* Architecture: [x86_64]
* nDPI version: [4.7.0]

**Comment by vel21ripn** [comment]
Fixed in 5f138c484891fd7c37be730f958fedbae78513b0

**Comment by netcons** [comment]
Thank you

**Comment by netcons** [comment]
I am sure this tested ok before, maybe I am missing something.

**Example 1:**
```
iptables -A FORWARD  -m ndpi --all --risk 1,27 -j DROP
iptables -vnxL | grep risk
       0        0 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            ndpi risk 0,27 proto all
```

**Example 2:**
```
iptables -A FORWARD  -m ndpi --all --risk 27 -j DROP
iptables -vnxL | grep risk
       0        0 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            ndpi risk 7 proto all
```

**Comment by vel21ripn** [comment]
Fixed in a408891

**Comment by netcons** [comment]
Thank you
